tests/test_file.py

```diff
@@ -100,44 +100,41 @@ def test(*args, **kwargs):
             return self.file_content
 
         monkeypatch.setattr('telegram.request.Request.retrieve', test)
-        out_file = file.download()
+        out_file: Path = file.download()
```

Comment:
why do you add these type annotations in the tests?

Reply:
i thought it would help readability, i'll remove them.

Reply:
thought it would be more readable like this. removed.